src/pluginmanager.cpp

```diff
@@ -124,6 +124,10 @@ PluginManager::eventHook(DmtcpEvent_t event, DmtcpEventData_t *data)
   case DMTCP_EVENT_RESTART:
   case DMTCP_EVENT_THREAD_RESUME:
 
+    if (event == DMTCP_EVENT_RESTART) {
+      DMTCP_RESTART_PAUSE(6);
```

Comment:
Do we need to use a descriptive variable name for 1-7 level? It is somewhat hard to understand which level stands for which in my opinion.

Reply:
@jiamingz9925 , where is a good place to document the levels 1-7, so that an end uer will see this?  I'm open to suggestions.

For now, I was assuming that the user would realize that level 1 is early in restart, and level 7 is late in restart, and then let them see the difference by trial and error.  Otherwise, there's a danger that any description becomes out-of-date as we update the code.